reg_tests/CTestList.cmake

```diff
@@ -147,10 +145,12 @@ add_test_r_np(periodic3dEdge 8)
 add_test_r(quad9HC 2)
 add_test_r(steadyTaylorVortex 4 6)
 add_test_r_rst2(steadyTaylorVortex_P4 8)
-add_test_r_rst2(hoVortex_P2 8)
 add_test_r(variableDensNonIso 2)
 add_test_r(variableDensNonUniform 2)
 add_test_r(variableDensNonUniform_P5 8)
+if(ENABLE_OPENFAST)
```

Comment:
**Wishlist**  It would be nice if the CTest output (as well as CDash) showed information about _skipped tests_ instead of just removing those tests.

Reply:
I was going to add this into this pull request but I could only find the following because documentation for CMake is hard to find: https://cmake.org/Wiki/CMake_Testing_With_CTest#Customizing_CTest and I didn't spend the time to test an implementation. I guess I just add the CTestCustom file, add all tests in CTestList, and then use `if(NOT ENABLE_OPENFAST)` to put the test in the `CTEST_CUSTOM_TESTS_IGNORE` list?

Reply:
@jrood-nrel @sayerhs  -- is this ready for merger, or is more work required?